Formula/libevent.rb

```diff
@@ -11,22 +11,32 @@ class Libevent < Formula
     sha256 "ef703db1b4cbdab35b89aabe80c225dd9b7a2c3ea14b1eae681478c5b9df15fe" => :yosemite
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/libevent/libevent", :using => :git
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "doxygen" => :build
-  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "openssl"
+  depends_on "openssl" => :recommended
```

Comment:
This should remain required as it has been ever since https://github.com/Homebrew/legacy-homebrew/pull/33282.